tasks/sign-index-image/sign-index-image.yaml

```diff
@@ -60,29 +59,28 @@ spec:
             '.sign.pipelineImage // .fbc.pipelineImage // $default_pipeline_image' ${DATA_FILE})
         config_map_name=$(jq -r '.sign.configMapName // .fbc.configMapName // "signing-config-map"' ${DATA_FILE})
         reference_image=$(params.referenceImage)
-        if [ -n "$(params.manifestDigestImage)" ]; then
-          manifestDigestImage="$(params.manifestDigestImage)"
-          manifest_digest="${manifestDigestImage#*@}"
-        else
-          manifest_digest="${reference_image#*@}"
-        fi
+
         pipelinerun_label="internal-services.appstudio.openshift.io/pipelinerun-uid"
 
         # Translate direct quay.io reference to public facing registry reference
         # quay.io/redhat/product----repo -> registry.redhat.io/product/repo
         reference_image=$(translate-delivery-repo $reference_image)
 
-        echo "Creating InternalRequest to sign image:"
-        echo "- reference=${reference_image}"
-        echo "- manifest_digest=${manifest_digest}"
-        echo "- requester=$(params.requester)"
+        # get all digests from manifest list
+        for manifest_digest in $(params.manifestListDigests)
```

Comment:
I asked this in https://github.com/hacbs-release/app-interface-deployments/pull/74 as well, but repeating here:

This looks like from now on we're only supporting images with a manifest list. Is that alright? Can we safely assume that there won't be any images with `"mediaType": "application/vnd.oci.image.manifest.v1+json"`?

Reply:
I addressed the comment in https://github.com/hacbs-release/app-interface-deployments/pull/74. The internal services call will fail if we do obtain any `manifest.v2+json` digests.

Reply:
Sounds good, thanks.